products/models.py

```diff
@@ -18,9 +17,17 @@ class Meta:
 class ProductImage(models.Model):
     product    = models.ForeignKey('Product', on_delete = models.CASCADE)
     image_urls = models.URLField()
+    
     class Meta:
         db_table = 'images'
 
+class ProductSizeJoined(models.Model):
```

Comment:
`ProductSize`만으로도 충분한 의미 전달이 될 것 같습니다!
```suggestion
class ProductSize(models.Model):
```